examples/v2/storage-bucket/storagebucket.jinja

```diff
@@ -0,0 +1,46 @@
+{#
+  20180717 - ohammami <osm.hammami@gmail.com>
+  version 0.1
+#}
+
+{% set ID = env['name'] %}
+
+resources:
+- name: {{ ID }}-bucket
```

Comment:
1.
I would advise not to use the `name` env variable for the bucket name, I would use a property.
2.
I would also remove the "bucket" string from the template and add it to the properties as well. ( Going further, I would write a helper function to generate the name of the bucket.)